scss/tech.scss

```diff
@@ -0,0 +1,67 @@
+.title-tech {
+  font-size: $text-medium;   
+  padding: 10px 10px 10px 10px;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+%size-tech {
+  width: 220px;
+  height: 300px;
+}
+
+%style-border {
+  @extend %size-tech;
+  border: thin solid $earls-green;
+  border-radius: 5px;
+  content: '';
+  position: absolute;
+  z-index: -100;    
+}
+
+.tech-panel {
+  @extend %size-tech;
+  align-items: center;
+  background-color: $earls-green;
+  border: solid $earls-green;
```

Comment:
There's no need to add a border of the same color as the background, the `border-radius` will suffice :smile_cat: